fdbrpc/FlowTransport.actor.cpp

```diff
@@ -977,6 +1012,7 @@ static void scanPackets(TransportData* transport,
                         const uint8_t* e,
                         Arena& arena,
                         NetworkAddress const& peerAddress,
+                        AuthorizedTenants authorizedTenants,
```

Comment:
It seems we are passing AuthorizedTenants per value for every packet multiple times, it would involve copying the set structure. We should pass it per reference if possible, or make it reference counted otherwise.

Reply:
this should've been a reference, I'll fix it